doc/src/release_notes.rst

```diff
@@ -20,8 +20,7 @@ To see the full list of changes check the list of `Git commits <https://github.c
 pgRouting 3.2.0 Release Notes
 -------------------------------------------------------------------------------
 
-To see all issues & pull requests closed by this release see the `Git closed milestone for 3.2.0
-<https://github.com/pgRouting/pgrouting/issues?utf8=%E2%9C%93&q=milestone%3A%22Release%203.2.0%22>`_ on Github.
+To see all issues & pull requests closed by this release see the `Git closed milestone for 3.2.0 <https://github.com/pgRouting/pgrouting/issues?utf8=%E2%9C%93&q=milestone%3A%22Release%203.2.0%22>`_ on Github.
```

Comment:
Need to add the issue about the version change on sphinx

Reply:
Ok, adding it